addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -187,4 +187,22 @@ options.registry.Parallax = options.Class.extend({
     },
 });
 
+options.userValueWidgetsRegistry['we-colorpicker'] = options.userValueWidgetsRegistry['we-colorpicker'].extend({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    toggleVisibility() {
+        this._super(...arguments);
+        if (this.$target.is('a')) {
+            // There is an option in link tools for that, do not duplicate it.
+            this.$el.hide();
+        }
+    },
```

Comment:
So, in master, we will drop this commit because we'll have the one in https://github.com/odoo/odoo/pull/108269, right ?

Reply:
Correct!